graylog2-server/src/main/java/org/graylog2/indexer/searches/SearchesClusterConfig.java

```diff
@@ -63,6 +63,7 @@
             .add("file")
             .add("source_file")
             .build();
+    private static final Set<String> DEFAULT_LINKED_SEARCH_FIELDS = Collections.emptySet();
```

Comment:
```text
graylog2-server/graylog2-server/src/main/java/org/graylog2/indexer/searches/SearchesClusterConfig.java:66: error: cannot find symbol
    private static final Set<String> DEFAULT_LINKED_SEARCH_FIELDS = Collections.emptySet();
```
This won't work unless you import the `Collections` class.

Reply:
Fixed!